_posts/2021-08-18-datafusion-5.0.0.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: post
+title: Apache Arrow DataFusion 5.0.0 Release
+date: "2021-08-18 00:00:00"
+author: pmc
+categories: [release]
+---
+<!--
+{% comment %}
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to you under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+The Apache Arrow team is pleased to announce the DataFusion 5.0.0 release. This covers 4 months of development work 
+and includes 211 commits from the following 31 distinct contributors.
+
+```
+$ git shortlog -sn 4.0.0..5.0.0 datafusion datafusion-cli datafusion-examples
+    61  Jiayu Liu
+    47  Andrew Lamb
+    27  Daniël Heres
+    13  QP Hou
+    13  Andy Grove
+     4  Javier Goday
+     4  sathis
+     3  Ruan Pearce-Authers
+     3  Raphael Taylor-Davies
+     3  Jorge Leitao
+     3  Cui Wenzheng
+     3  Mike Seddon
+     3  Edd Robinson
+     2  思维
+     2  Liang-Chi Hsieh
+     2  Michael Lu
+     2  Parth Sarthy
+     2  Patrick More
+     2  Rich
+     1  Charlie Evans
+     1  Gang Liao
+     1  Agata Naomichi
+     1  Ritchie Vink
+     1  Evan Chan
+     1  Ruihang Xia
+     1  Todd Treece
+     1  Yichen Wang
+     1  baishen
+     1  Nga Tran
+     1  rdettai
+     1  Marco Neumann
+```
+
+<!--
+$ git log --pretty=oneline 4.0.0..5.0.0 datafusion datafusion-cli datafusion-examples | wc -l
+     211
+-->
+
+The release notes below are not exhaustive and only expose selected highlights of the release. Many other bug fixes 
+and improvements have been made: we refer you to the complete 
+[changelog](https://github.com/apache/arrow-datafusion/blob/master/datafusion/CHANGELOG.md).
```

Comment:
```suggestion
[changelog](https://github.com/apache/arrow-datafusion/blob/5.0.0/datafusion/CHANGELOG.md).
```

Might make sense to pin to the 5.0.0 tag release so it doesn't get stale when we do another release